SDC Decomposition Sprint 4 (#1406)
CodeYourFuture/curriculum: common-content/en/module/decomposition/dataclasses/index.md

+++
title = "Dataclasses"
time = 30
emoji= "📝"
objectives = [
  "Define a value type.",
  "Use `@dataclass` in Python to make a value type.",
  "Customise a `@dataclass` in Python with extra methods."
]

[build]
  list = "local"
  publishResources = false
  render = "never"
+++

We've seen that grouping together fields and methods into a class can help us encapsulate them.

> 

Reviewer: ```suggestion
```

> Often when programming we define a class whose purpose is just to group together related data, and provide access to it.

Reviewer: ```suggestion
We can define a class whose purpose is just to group together related data, and provide access to it.
```
